README.md

```diff
@@ -8,17 +8,19 @@
 
 Follow instructions here: https://github.com/openmrs/openmrs-distro-referenceapplication/tree/3.x
 
-#### Open port `3306` + enable the bin log:
-If running in Docker, you can do it by setting the `ports` and overriding the `command`:
+#### Enable mysql bin log
+IF using docker-compose, you can do it by exposing the `port` and overriding the `command` parameters for example;
```

Comment:
If using Docker Compose, expose the `port` and override the `command` parameters in **docker-compose.yml**. For example: